MPC/config/dafcompilersupport.mpb

```diff
@@ -35,6 +35,12 @@ feature(!warn_unused_variable){
   }
 }
 
+feature(no_deprecated_declarations){
+  verbatim(gnuace,top,1) { // NOTE: explicit ,1 in verbatim add = 1 means its appended
```

Comment:
This is a pretty heavy-handed way of going about this. I would want to make sure that as further ACE-TAO releases come out that we revisit this one.